requirements.txt

```diff
@@ -12,3 +12,5 @@ django-modelcluster>=2.0,<3.0
 psycopg2
 gunicorn
 django_compressor==2.0
+celery<4.0
```

Comment:
do we still need django-celery if we're using the newer version 4? http://docs.celeryproject.org/en/latest/django/first-steps-with-django.html